src/panel/plugin/plugin-manager.c

```diff
@@ -0,0 +1,472 @@
+/*
+ * This file is part of budgie-desktop
+ *
+ * Copyright Budgie Desktop Developers
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+
+#include "plugin-manager.h"
+
+#include "budgie-config.h"
+#include "applet.h"
+#include "plugin.h"
+
+#include <gobject-introspection-1.0/girepository.h>
+#include <libpeas.h>
```

Comment:
Here we #include differently https://github.com/BuddiesOfBudgie/budgie-desktop/pull/715/files#diff-8099e7e0bcf95b8b2c50ce8111790e7632f31c816c07c5ffb6989243c6f385c0R16